File: doc/_i18n/jp/tutorials/introduction/ef-controller/update_ef_task.py
# 現在の目標を取得する
pt = efTask.get_ef_pose()
# 回転と位置の目標を更新する
efTask.set_ef_pose(sva.PTransformd(sva.RotY(-math.pi/2), eigen.Vector3d(0.5, -0.5, 1.2)))

File: doc/_i18n/en/tutorials/introduction/ef-controller/update_ef_task.cpp
// Get the current objective
auto pt = efTask->get_ef_pose();
// Update the rotation and position objective
efTask->set_ef_pose(sva::PTransformd{sva::RotY(-M_PI/2), Eigen::Vector3d{0.5, -0.5, 1.2}});
